.gitignore

```diff
@@ -2,4 +2,5 @@
 coverage/
 cypress/
 dist/
+ts-dist/
 node_modules/
```

Comment:
Sorry for the nitpick, but can you put it below `node_modules`? The entries are sorted alphabetically